josm-presets/de-signals-eso.xml

```diff
@@ -2244,6 +2244,44 @@ See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 					de.text="Frequenz (falls angezeigt)"
 					/>
 			</item>
+			<item name="Isolated overlap signals" de.name="Elektrische Streckentrennung" icon="de/el-begin-isolated-overlap-30.png" type="node">
+				<label text="Isolated overlap signs" de.text="Elektrische Streckentrennung" />
+				<label text="This signal is being mapped as a node on the track." de.text="Wird als Punkt auf dem Gleis erfasst." />
+				<space />
+				<key key="railway"
+					value="signal" />
+				<check key="railway:signal:electricity:deactivated"
+					text="Out of order"
+					de.text="Ungültigkeitskreuz"
+					/>
+				<combo key="railway:signal:position"
+					text="Signal position"
+					de.text="Signalstandort"
+					values="left,right,overhead"
+					display_values="left in direction of OSM way,right in direction of OSM way,overhead (at catenary level)"
+					de.display_values="Links von Wegrichtung,Rechts von Wegrichtung,in der Oberleitung"
+					/>
+				<reference ref="direction_fw_bw" />
+				<space />
+				<key key="railway:signal:electricity"
+					value="DE-ESO:streckentrennung" />
+				<combo key="railway:signal:electricity:type"
+					text="Exact type"
+					de.text="Signaltyp"
+					values="begin_of_isolated_overlap;end_of_isolated_overlap"
+					display_values="Begin of isolated overlap,End of isolated overlap"
+					de.display_values="Beginn elektrische Streckentrennung,Ende elektrische Streckentrennung"
```

Comment:
You use different delimiters. This is raised by JOSM.

```sh
michael@ortelius:/tmp$ josm
[…]
2024-08-29 17:48:28.614 SCHWERWIEGEND: Defekte Taggingvorlage "railway:signal:electricity:type-Exact type" - Anzahl der Elemente in 'display_values' muss gleich sein wie in 'values'.
2024-08-29 17:48:28.614 SCHWERWIEGEND: Detaillierte Information: [Begin of isolated overlap, End of isolated overlap] <> [begin_of_isolated_overlap;end_of_isolated_overlap]
2024-08-29 17:48:28.614 SCHWERWIEGEND: Defekte Taggingvorlage "railway:signal:electricity:type-Exact type" - Anzahl der Elemente in 'locale_display_values' muss gleich sein wie in 'values'.
2024-08-29 17:48:28.614 SCHWERWIEGEND: Detaillierte Information: [Beginn elektrische Streckentrennung, Ende elektrische Streckentrennung] <> [begin_of_isolated_overlap;end_of_isolated_overlap]
```

Reply:
Thanks. Now it should work. 
Rendering inside JOSM works: 
![image](https://github.com/user-attachments/assets/ba386c94-08c0-4260-be85-6792a3ae390a)